docker-compose.dev.yml

```
@@ -47,5 +47,11 @@ services:
       - ./certbot/www:/var/www/certbot
     networks: [backend]
 
+  redis:
+    image: redis:7
+    container_name: redis
+    restart: always
+    networks: [ backend ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)